src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DefaultRazorEditorFactoryService.cs

```diff
@@ -49,7 +48,7 @@ public override bool TryGetDocumentTracker(ITextBuffer textBuffer, out VisualStu
                 return false;
             }
 
-            if (!textBuffer.Properties.TryGetProperty(typeof(VisualStudioDocumentTracker), out documentTracker))
+            if (!textBuffer.Properties.TryGetProperty(typeof(VisualStudioDocumentTracker), out documentTracker!))
```

Comment:
Err, what? This is going to potentially output a null from a method that is clearly annotated that it won't do that.

Reply:
`TryGetProperty` comes from an external assembly (in this case `MS.VS.CoreUtility`), that is not properly null-annotated, so I have to do a suppression here

Reply:
Certainly can't change the null annotation on the method, but suppression still just hides bugs IMO. If someone is investigating a crash or NFW, and are having to reverse engineer which something could be null, then they want to be able to trust the annotations as written, and these sorts of suppressions are very hidden, even if someone does look into this method. This seems a lot safer to me:
```suggestion
            if (!textBuffer.Properties.TryGetProperty(typeof(VisualStudioDocumentTracker), out documentTracker) ||
                documentTracker is null)
```